.github/workflows/pr-validation.yml

```
@@ -18,87 +20,83 @@
         echo "🔍 Validating Git-Flow compliance..."
         echo "Source branch: ${{ github.head_ref }}"
         echo "Target branch: ${{ github.base_ref }}"
         
-        # Block direct PRs to main (should come from develop)
-        if [[ "${{ github.base_ref }}" == "main" && "${{ github.head_ref }}" != "develop" ]]; then
-          echo "❌ Direct PRs to 'main' are not allowed!"
-          echo "💡 Follow git-flow: merge to 'develop' first, then create release PR from 'develop' to 'main'"
-          exit 1
+        # Validate Git-flow branch targeting patterns
+        echo "🔍 Source: ${{ github.head_ref }} → Target: ${{ github.base_ref }}"
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)